doc/release/migration_guide_from_1.x_to_2.0.rst

```diff
@@ -185,3 +188,28 @@ networkx namespace. These are:  ``G.add_path``, ``G.add_star``, ``G.add_cycle``,
 ``G.number_of_selfloops``, ``G.nodes_with_selfloops``, and ``G.selfloop_edges``.
 These are replaced by ``nx.path_graph(G, ...)`` ``nx.add_star(G, ...)``,
 ``nx.selfloop_edges(G)``, etc.
+
+-------
+
+Writing code that works for both versions
```

Comment:
A couple other things you may want to say something about here:

I had to change
```
graph.node[copy_id] = graph.node[node_id]
```
to
```
graph.node[copy_id].update(graph.node[node_id])
```

I suppose you should say something about ``node`` here and that it will eventually be removed.

I had to add ``fresh_copy(self)`` to get ``copy`` to work correctly.

I also changed ``G.node.items()`` to ``G.nodes(data=True)``.